buildSrc/src/main/java/gradlebuild/NcursesPlugin.java

```diff
@@ -14,13 +15,18 @@
 import org.gradle.platform.base.PlatformContainer;
 
 import java.io.File;
+import java.util.List;
+import java.util.SortedSet;
+import java.util.stream.Collectors;
 
 import static gradlebuild.NativeRulesUtils.addPlatform;
 
 public class NcursesPlugin extends RuleSource {
 
-    @Model NcursesVersion ncursesVersion() {
-        return new NcursesVersion(inferNCursesVersion());
+    @Model List<NcursesVersion> ncursesVersions() {
+        return inferNCursesVersions().stream()
+            .map(NcursesVersion::new)
+            .collect(Collectors.toList());
```

Comment:
Nit: would be nicer if `inferNCursesVersions()` would already return `NcursesVersion` objects.

Converting between a `List` and a `SortedSet` also looks unnecessary. Can we just merge these?

Reply:
I changed the usage of `NcursesVersion`. PTAL!